schemas/documents/terrasos/complianceInfo.js

```diff
@@ -0,0 +1,19 @@
+export default {
+  name: 'complianceInfo',
```

Comment:
could you add this to the list in `uniqueDocuments`?
I think we might be missing `tebuBanner` there as well